src/main/java/me/figo/models/Icon.java

```diff
@@ -0,0 +1,30 @@
+package me.figo.models;
+
+import java.util.Map;
+
+import com.google.gson.annotations.Expose;
+
+public class Icon {
+	
+	@Expose
+	private Map<String,String> resolutions;
+	
+	@Expose
+	private String url;
+
+	public Map<String, String> getResolutions() {
+		return resolutions;
+	}
+
+	public void setResolutions(Map<String, String> resolutions) {
+		this.resolutions = resolutions;
+	}
+
+	public String getUrl() {
+		return url;
+	}
+
+	public void setUrl(String url) {

```

Comment:
What are the setters good for? The model is only used to serialize a JSON response, isn't it? It is not possible to alter an icon through the API. The same holds true for the `setResolutions`-method.

Reply:
I'll merge this anyhow, since it seems to be time critical to have a working release for the partners.